psl-core/src/test/java/org/linqs/psl/application/inference/MPEInferenceTest.java

```diff
@@ -195,4 +197,68 @@ public void testLogicalTautologyTrivial() {
         mpe.close();
         inferDB.close();
     }
+
+    /**
+    * Make sure that skipAtomCommit flag from CLI works and no atom is persisted in the database.
+    */
+    @Test
+    public void testSkipAtomCommit() {
+        // Arrange
+        TestModel.ModelInformation info = TestModel.getModel();
+        Set<StandardPredicate> toClose = new HashSet<StandardPredicate>();
+        Database inferDB = info.dataStore.getDatabase(info.targetPartition, toClose, info.observationPartition);
+        MPEInference mpe = new MPEInference(info.model, inferDB);
+                
+        float preInferenceTotalValue = 0.0f;
+        for (RandomVariableAtom atom : inferDB.getAllGroundRandomVariableAtoms(info.predicates.get("Friends"))) {
+            preInferenceTotalValue += atom.getValue();
+        }
+
+        // Act
+        mpe.inference(true);
+        
+        mpe.close();
+        inferDB.close();
+        
+        inferDB = info.dataStore.getDatabase(info.targetPartition, toClose, info.observationPartition);
```

Comment:
Command this line and explain what is happening.
Remember, other people may look at this test later.

Reply:
Unresolved comment.